### ISSUE TITLE
Identify returning /mnt/c/users on Windows instead of C:\Users

### ISSUE THREAD

**Issue by TheJaredWilcurt** [saschanaz/flif-wasm#7]
Using version `1.0.6`.

* * *

**Running this:**
```
flif.exe -d -i "C:\Users\Owner\Desktop\cat.flif"
```
**Returns:**
```
C:\Users\Owner\Desktop\cat.flif: FLIF image, 80x64, 8-bit RGBA, non-interlaced
```
**However, running this:**
```js
var exec = require('child_process').execSync;
var flif = require.resolve('flif-wasm');

var executableAndArgs = 'node "' + flif + '" -d -i "C:\Users\Owner\Desktop\cat.flif"';
var child = exec(executableAndArgs);

var response = child.toString().trim();
console.log(response);
```
**Returns:**
```
/mnt/c/Users/Owner/Desktop/cat.flif: FLIF image, 80x64, 8-bit RGBA, non-interlaced
```

**Comment by saschanaz** [comment]
Ah, thank you for opening this, I forgot updating this repo with NODERAWFS.

**Comment by saschanaz** [comment]
Should be fixed by https://github.com/saschanaz/flif-wasm/commit/dd4371d94f5da243d6970cf84f5666815122b76f, released as 1.0.7.

**Comment by TheJaredWilcurt** [comment]
Verified issue no longer occurs. :+1:
